src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -92,21 +92,21 @@ public Manufacturer update(Manufacturer manufacturer) {
 
     @Override
     public boolean delete(Long id) {
-        String query = "UPDATE manufacturers SET deleted = TRUE WHERE id = ?";
+        String query = "UPDATE drivers SET is_deleted = TRUE WHERE driver_id = ?";
```

Comment:
why did you change manufacturer to dirver?

Reply:
omg, my fault(